changelog/unreleased/bugfixes/6986.md

```diff
@@ -0,0 +1,9 @@
+- Improved duration calculation for EV routes to account for charging time.
+- Added snapping feature to the navigation for walking and cycling profiles.
```

Comment:
```suggestion
```
Is that related to native reroute controller? If yes, we probably don't need it.

Reply:
It's customer-related, I think we should leave it.
See `mapbox-navigation-native/pull/6741`.